sectorbuild_cgo.go

```diff
@@ -137,6 +138,42 @@ func (sb *SectorBuilder) ReadPieceFromSealedSector(sectorID uint64, offset uint6
 func (sb *SectorBuilder) SealPreCommit(ctx context.Context, sectorID uint64, ticket SealTicket, pieces []PublicPieceInfo) (RawSealPreCommitOutput, error) {
 	fs := sb.filesystem
 
+	cacheDir, err := sb.sectorCacheDir(sectorID)
```

Comment:
I would move this checking logic into a separate function, just to keep it clean

Reply:
hrm... on second thought, we're using these values later in the function. This is fine for now